ui/src/App.tsx

```diff
@@ -1,25 +1,95 @@
-import React from "react";
-import logo from "./logo.svg";
-import "./App.css";
+import { Note } from "@mui/icons-material";
+import {
+  AppBar,
+  Box,
+  Button,
+  Card,
+  CardActions,
+  Input,
+  Toolbar,
+  Typography,
+} from "@mui/material";
+import * as React from "react";
 
 function App() {
+  const [showDetailedInput, setShowDetailedInput] =
+    React.useState<boolean>(false);
+
+  function handleNewNoteClick() {
+    setShowDetailedInput(true);
+  }
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Box>
+      <AppBar position="static" style={{ background: "#808080" }}>
+        <Toolbar disableGutters>
+          <Note
+            fontSize="large"
+            sx={{ display: { xs: "flex" }, mr: 1, ml: 1.5 }}
```

Comment:
You are not using RWD for this one. We can simplify it.

```suggestion
            sx={{ display: "flex", mr: 1, ml: 1.5 }}
```